README.rst

```diff
@@ -11,20 +11,55 @@ into this repository. The translation files in this repository can then be acces
 using the `openedx-atlas`_ CLI tool to download specific directories of translation files
 from openedx-translations.
 
-Current State
-*************
+This repository implements the `OEP-58`_ proposal.
 
-This repository is currently under active development with limited use.
-The GitHub Action
-``extract-translation-source-files.yml`` generates English translation source
-files for the configured repositories. These translation source files are the only
-ones uploaded to the Transifex project openedx-translations. The English translation
-source files have only been translated into one language: French Canadian (fr_CA). The
-`openedx-atlas`_ CLI tool can only be used to pull translation files from the credentials
-directory in openedx-translations.
+Main and Release branches
+*************************
+
+This repository has a main branch in addition to a dedicated branch for every
+release. As of May 10th, 2024 the following are the release branches:
+
+- `main branch <https://github.com/openedx/openedx-translations>`_:
+
+  This branch is used for the latest version of Open edX such as
+  `Tutor nightly`_, `edx-platform "master" branch`_ and others.
+
+  To translate the latest versions the `open-edx/openedx-translations`_ Transifex
+  project should be used.
+
+
+- `open-release/redwood.master branch <https://github.com/openedx/openedx-translations/tree/open-release/redwood.master>`_:
+
+  This branch is used for the latest version of Open edX such as
+  `Tutor Redwood v18`_, `edx-platform "open-release/redwood.master" branch`_
+  and others.
+
+To translate the latest versions the `open-edx/openedx-translations-redwood`_
```

Comment:
```suggestion
To update translations for the Redwood release, the `open-edx/openedx-translations-redwood`_
```

I think this is important because it's not the "latest version", it's the Redwood one. Also, could you indent this as above?